src/dstack/_internal/core/models/configurations.py

```diff
@@ -267,6 +276,16 @@ def convert_replicas(cls, v: Any) -> Range[int]:
             )
         return v
 
+    @validator("gateway")
+    def validate_gateway(
```

Comment:
Is this for the sake of human-readable error message? Otherwise, `Optional[Union[Literal[False], str]]` would do the same validation automatically (didn't check though)

Reply:
I tried `Union[Literal[False], str]` first, but this way `true` passes validation and is converted to `"true"`, which is not expected